zerver/data_import/mattermost.py

```diff
@@ -89,6 +97,9 @@ def get_full_name(user_dict: Dict[str, Any]) -> str:
     if is_team_admin(user_dict):
         role = UserProfile.ROLE_REALM_OWNER
 
```

Comment:
I think that stylistically we don't want an empy line above the `elif`, it'll be more consistent with how we usually structure these blocks not to have it